game/ui/city/buildingscreen.cpp

```diff
@@ -31,17 +31,15 @@ std::shared_future<void> loadBattleBuilding(sp<GameState> state, sp<Building> bu
                                             StateRef<Vehicle> playerVehicle)
 {
 	auto loadTask = fw().threadPoolEnqueue(
-	    [hotseat, building, state, raid, playerAgents, playerVehicle]() -> void {
+	    [hotseat, building, state, raid, playerAgents, playerVehicle]() mutable -> void {
```

Comment:
This should probably be pulled out into another commit

Reply:
I've split this between commits 02bc978 and 439288c